src/Traits/Translatable.php

```diff
@@ -244,6 +245,81 @@ public function setAttributeTranslations($attribute, array $translations, $save
         return $response;
     }
 
+    /**
+     * Gets Entry if has translated field field with value via scope.
+     * 
+     * @example  $class->whereTranslation('de', 'title', 'zuhause') 
+     * 
+     * @param builder       $query
+     * @param string|array  $locale     {required} locale(s) you are looking for the field.
+     * @param string        $field      {required} the field your looking to find a value in.
+     * @param string        $operator   {required} value you are looking for or a relation modifier.
+     * @param string        $value      {optional} value you are looking for. Only use if you supplied an operator.
+     * @param bool          $default    {optional} fi true checks for $value is in default language before checking translations.
+     *
+     * @return Builder|null
+     * 
+     */
+    public function scopeWhereTranslation(Builder $query, $locales, string $field, string $operator, string $value = null, bool $default = false){
+        return $query->whereTranslation( $locales, $field, $operator, $value, $default );
+    }
+
+    /**
+     * Gets Entry if has translated field with value.
+     * 
+     * @example  Class::whereTranslation(['de', 'iu'], 'title', 'zuhause', '=', true)
+     * 
+     * @author reageek <https://github.com/reageek> && emptynick <https://github.com/emptynick>.
+     * @todo refactor out of static if you wish.
+     * 
+     * @param string|array  $locale     {required} locale(s) you are looking for the field.
+     * @param string        $field      {required} the field your looking to find a value in.
+     * @param string        $operator   {required} value you are looking for or a relation modifier such as LIKE, =, etc.
+     * @param string        $value      {optional} value you are looking for. Only use if you supplied an operator.
+     * @param bool          $default    {optional} if true checks for $value is in default database before checking translations.
+     *
+     * @return Builder|null
+     * 
+     */
+    public static function whereTranslation( $locales, string $field, string $operator, string $value = null, bool $default = false )
+    {
+        if( !is_array( $locales )){
+            $locales = [$locales];
+        }
+        if ( !isset( $value ) ) {
+            $value = $operator;
+            $operator = '=';
+        }
+        if($default){
+            $default = self::where( $field, $value );
+            if( $default->first() ){
+                //Found in default database model
+                return $default;
+            }
+        }
+        $self = new static;
+        $table = ( isset($self->table) ) ? $self->table : studly_case( str_plural ( class_basename( $self ) ) ); //discern table name
```

Comment:
`$self->getTable()` does this for you already
```suggestion
        $table = $self->getTable();
```